@here/harp-mapview/lib/poi/PoiManager.ts

```diff
@@ -590,165 +579,4 @@ export class PoiManager {
 
         return textElement;
     }
-
```

Comment:
These two functions below getRenderStyle / getLayoutStyle were duplicated in TileGeometryCreator. I've moved them to TextStyleCache.